packages/compartment-mapper/src/bundle.js

```diff
@@ -239,10 +244,21 @@ export const makeBundle = async (read, moduleLocation, options) => {
   const parsersInUse = new Set();
   for (const module of modules) {
     module.indexedImports = Object.fromEntries(
-      Object.entries(module.resolvedImports).map(([importSpecifier, key]) => [
-        importSpecifier,
-        modulesByKey[key].index,
-      ]),
+      Object.entries(module.resolvedImports).map(([importSpecifier, key]) => {
+        key = aliases.get(key) ?? key;
+        const module = modulesByKey[key];
+        if (module === undefined) {
```

Comment:
So the original implementation would fail to read the index if an alias was used?

Reply:
Indeed, the modules array lacks entries for the aliases.